### ISSUE TITLE
[Question] Is gif and direct link supported?

### ISSUE THREAD

**Issue by feruzm** [chjj/marked#771]
Right now I get images to display when it is properly markdown, but if it is gif or direct link, it won't show.

Is there option to enable direct image link to display?

For example, if user paste link and link is for image, it will display when it is marked back to html ?
I can make small changes or enable options, if there is any. I just started to use this, it is super fast and great! Thank you for this!

**Comment by Feder1co5oave** [comment]
Sorry Feruz, your question is not very clear. Can you rephrase?

I have no problems including directly linked images:

`![Pulsar](https://upload.wikimedia.org/wikipedia/commons/0/07/Game_of_life_pulsar.gif)`
->
`<p><img src="https://upload.wikimedia.org/wikipedia/commons/0/07/Game_of_life_pulsar.gif" alt="Pulsar"></p>`

![Pulsar](https://upload.wikimedia.org/wikipedia/commons/0/07/Game_of_life_pulsar.gif)

**Comment by feruzm** [comment]
Hi @Feder1co5oave ,
Yes, I got it working, I meant. If you were to put just a link without formatting `[]()` for images would it parse them correctly. But solution I used was, after filtering links, I filter link to check if it is image and used above markdown as you mentioned.
